catalog/dags/data_refresh/indexer_worker/indexer_worker/elasticsearch_models.py

```diff
@@ -0,0 +1,335 @@
+"""
```

Comment:
Copied from `ingestion_server` without changes to logic. This provides the mappings from db record to ES document.